packages/contracts-bedrock/src/L2/L1BlockInterop.sol

```diff
@@ -109,10 +72,6 @@ contract L1BlockInterop is L1Block {
 
         if (_type == ConfigType.SET_GAS_PAYING_TOKEN) {
```

Comment:
Same here, are we expecting to add new members to `ConfigType`?

Reply:
Is a good question we could ask to them

Reply:
It is how it works, `OptimismPortal.setConfig()` can send multiple types, so this has to be extensible.

We will be using this feature in another PR so it should stay like this

Reply:
Mm... I think that the enum is right cause it will be easier to expand if needed... but this is no needed cause if you have to add another member you have to modify the function too.

Reply:
Without the if any call to `setConfig` means calling `_setGasPayingToken` and it shouldn't be like that